### DISCUSSION TITLE
DetailView Config files doubts

### DISCUSSION THREAD

**Discussion by BlueRyse** [noi-techpark/it.bz.opendatahub.databrowser#121] [General]
Hello everyone,

Me and Lukas wanted to start working on the config files for the detail views and we were following the provided design sheet file, but we are confused on some conflicts between the Repository and the file.

In the Repository, the Detail views are configured with one file per dataset inside /src/tourism/config/DatasetName.

Instead, in the file:

- Columns A-M represents categories with all the attributes from every dataset, but we didn't understand where these categories should be configured in the code and how.

- Columns M-Y represents special cases/attributes per dataset, but most of them are not present in the Columns A-M and we didn't understand why that is the case, and how these attributes should be handled in the code.

Can you give us a bit more information on how to structure the detail views for the Datasets?

**Comment by gappc** [comment]
Hi @BlueRyse,

@pkritzinger answered the questions by email some days ago. In case you missed it, here is the content: @Paul

> Hi Sergio,
>
> I’ll shortly try to describe the idea behind the configuration of the detail view:
>
> * Columns A-M refer to categories and attributes that can be found in multiple datasets. 
> We thought that it would be efficient to create a standard configuration for categories and attributes in Column A-M. If the user has a look at a specific dataset, only those attributes should be shown that are also available within the API of the respective dataset.
> * Columns N-Y are special cases/ attributes per dataset.
> This is why you will not find any main data, text information… within the column POI Details.
>
> Conclusion:
> By using the same configuration for columns A-M we make sure that datasets are always structured in the same way (of course it depends on the attributes provided, but it should be an improvement to the current databrowser where attributes are not always assigned to the same categories). And users should learn rather quickly that they can access dataset-specific attributes by means of the category “#Dataset-name#-Details”.
>
> Hope this helps?
>
> Do not hesitate to drop me a line in case of questions.
>
> Best,
> Paul

Does that information help you?

**Comment by ila131** [comment]
Hello everyone,
Delia and I have some doubts about column C and D of the Excel file regarding the detail view. The linked screenshots are not very clear. Where and how should we implement these changes?

Could you give us a more specific explanation about the changes you wanted to apply?

Thank you in advance

Best,
Ilaria

**Comment by pkritzinger** [comment]
@ila131 
Column C:
The order and/ or categorization of attributes remains the same as it is now. I only inserted the screenshot because I always forgot how the component in the current databrowser looks like :)

Column D:
In this column all attributes allowing the integration (in case of edit view)/ visualization (in case of detail view) of related content are put together. Also here the screenshot has only visual supportive character.
